proto/otaclient_iot_logging_server_v1.proto

```diff
@@ -14,38 +14,50 @@
 
 syntax = "proto3";
 
-service OtaClientLoggingService {
+service OtaClientIoTLoggingService {
   /*
    * `PutLog` service requests OTA Client logging service to put log.
    */
   rpc PutLog(PutLogRequest) returns (PutLogResponse) {}
+
+  /*
+   * `Check` requests OTA Client logging service to check the health of the
+   * service.
+   */
+  rpc Check(HealthCheckRequest) returns (HealthCheckResponse);
```

Comment:
This follows https://github.com/grpc/grpc/blob/master/src/proto/grpc/health/v1/health.proto